narayana-bom/pom.xml

```diff
@@ -294,7 +219,7 @@
         <version>${version.jakarta.xml.ws.jakarta.xml.ws-api}</version>
       </dependency>
 
-      <!-- Narayan full -->
+      <!-- Narayana full -->
```

Comment:
This one goes in a different commit

Reply:
Thanks Manuel, I agree and I will update my PR.